Services/DataX.Config/DataX.Config/InternalService/FlowDataManager.cs

```diff
@@ -33,16 +34,28 @@ public FlowDataManager(ConfigGenConfiguration configuration, IDesignTimeConfigSt
         private ConfigGenConfiguration Configuration { get; }
         private ICommonDataManager CommonsData { get; }
 
+        private string GetFlowConfigName(string type)
+        {
+            if (!string.IsNullOrEmpty(type) && (type == Constants.InputType_Kafka || type == Constants.InputType_KafkaEventHub))
+            {
+                return CommonDataName_KafkaFlowConfig;
+            }
+
+            return CommonDataName_DefaultFlowConfig;
+        }
+
         public async Task<FlowConfig> GetByName(string flowName)
         {
             var json = await this.Storage.GetByName(flowName, DataCollectionName);
             return FlowConfig.From(json);
         }
 
-        public async Task<FlowConfig> GetDefaultConfig(TokenDictionary tokens = null)
+        public async Task<FlowConfig> GetDefaultConfig(string inputType = null, TokenDictionary tokens = null)
```

Comment:
the `GetDefaultConfig` is supposed to get default template without inputs. If we need to get config for different type of input, let's have a `GetConfigByInputType` instead.

Reply:
updated. thanks.